### PR TITLE
[MDS-6017] Upgraded Syncfusion to latest version in Minespace and Core to fix white screen when closing PDF viewer

### PR DESCRIPTION
**Pull request by simensma-fresh**
## Objective 

[MDS-6017](https://bcmines.atlassian.net/browse/MDS-6017)

Upgraded Syncfusion to latest version in Minespace and Core. This fixes an error when closing a PDF viewer in core through the MMS file explorer

**Move to common package**
- Moved all Syncfusion related functionality to the common package 
- Converted PDF viewer component to TS / functional component

**Use instance method to open PDF viewer**
- Use `modal.info to open the modal the PDF viewer is in instead of the `<Modal` component, this fixed an error that sometimes popped up

**Upgrade Syncfusion**
- The issue preventing an upgrade of Syncfusion in minespace has been fixed: The `$theme` variable we have set to `default` need to be `Material` when including the Syncfusion scss, otherwise things break 👀

### CODE REVIEW

**Review by taraepp — DISMISSED** [review]

File: services/minespace-web/src/styles/tools/syncfusion.scss [hunk 1]
```diff
@@ -29,27 +29,14 @@ $grey-dark-font: #ffffff;
...
-@import "~@syncfusion/ej2-react-pdfviewer/styles/material.scss";
-@import "~@syncfusion/ej2-react-filemanager/styles/material.scss";
-// NOTE: We may need these dependencies in the future if we implement the document/spreadsheet viewer
-// @import "../../../node_modules/@syncfusion/ej2-react-documenteditor/styles/material.scss";
-// @import "../../../node_modules/@syncfusion/ej2-react-spreadsheet/styles/material.scss";
-@import "~@syncfusion/ej2-notifications/styles/material.scss";
+// Syncfusion requires an undocumented $theme variable to be set in order to import the correct theme.
+// This is a workaround to allow us to use the Material theme while still being able to use a different
+// $theme variable elsewhere in the application.
+@use 'syncfusion_imports' with ($theme: 'Material');
```
Comment (matbusby-fw) [inline services/minespace-web/src/styles/tools/syncfusion.scss:35]: Oh man.  So happy you found this.

**Review by matbusby-fw — DISMISSED** [review]
Well, it's nice to see that almost everything I was trying to do worked in the end.  That `theme` variable is frustrating, but at least you figured it out!

**Review by henryoforeh-dev — APPROVED** [review]

File: services/minespace-web/webpack.config.ts [hunk 2]
```diff
@@ -38,10 +38,11 @@ const PATHS = {
 };
 
 const BUILD_FILE_NAMES = {
-  css: "style/[name].[contenthash:4].css",
-  bundle: "js/bundle.[chunkhash:4].js",
-  vendor: "js/[id].[chunkhash:4].js",
-  assets: "assets/[name].[hash:4].[ext]",
+  css: "style/[name].[contenthash:8].css",
+  cssChunkName: "style/[name].[chunkhash:8].css",
```
Comment (simensma-fresh) [inline services/minespace-web/webpack.config.ts:42]: Changing these from `chunkhash:4` / `chunkhash:8` fixed the webpack build issue that popped up 🤷

Comment (simensma-fresh) [inline services/minespace-web/webpack.config.ts:42]: Changing these from `chunkhash:4` / `chunkhash:8` fixed the webpack build issue that popped up 🤷

Comment (matbusby-fw) [reply services/minespace-web/webpack.config.ts:42]: 🙃 Obviously.  *EVERYONE* knows not to use `chunkhas:4`

**Review by matbusby-fw — APPROVED** [review]

**Pull request merged**